app/src/NativeBase/Containers/ProfileContainer.tsx

```diff
@@ -13,20 +44,72 @@ import React from 'react'
 const ProfileContainer = () => {
   return (
     <VStack
-      paddingTop="4"
-      paddingBottom="2"
-      alignItems="center"
-      space="4"
-      backgroundColor="secondaryGrey.60"
+      padding={2}
+      backgroundColor="bg.100"
       _dark={{ backgroundColor: 'bgDarkMode.100' }}
     >
-      <Center
-        backgroundColor="bg.100"
-        _dark={{ backgroundColor: 'bgDarkMode.100' }}
-        width="100%"
-      >
-        <Heading size="sm">My Profile</Heading>
+      <Center>
+        <Heading fontFamily="title" size="lg">
+          My Profile
+        </Heading>
+        <Box
+          marginTop={6}
+          height="93px"
+          width="90px"
+          borderColor="darkGrey.100"
+          borderWidth="4px"
+          borderRadius="full"
+        />
+        <HStack>
+          <Heading fontSize="md" fontWeight="700">
+            Full Name
+          </Heading>
+          <Text fontSize="xs" paddingLeft="-2">
+            (She)
+          </Text>
+        </HStack>
+        <Text fontSize="xs" marginTop="-4">
```

Comment:
It looks like there are several negative `paddingTop` and `marginTop` values here -- is there a better alternative to having to set this repeatedly for each element?  E.g. could the `Center` be a `VStack` instead and then set the `spacing` property on that?

Reply:
@joefhall yes i have used negative margins on my feature My Experience likewise. I did experiment with using various components but the way i ended up doing this was the only way i could find to get both logo aligned with the Header text and avoid it either disappearing from the screen or wrapping the words one or top of the other. I think if there's a better way to do this on my PR, a few pointers might be helpful given the length of time i spent on it y'day (if that's okay)!

Reply:
Hi @joefhall, I was counting on you picking this up :) Been wanting to bounce this off you. 

Many of the `marginTop`, `paddingTop` and `paddingLeft` values are actually negative. This is due to the fact that the `Text` component in the `StaTheme` has default padding of 2px. Adding a `space` property of "-2" to the parent component would counteract that, but it would increase the need for additional margin in other places. The result might be more confusing.

The best solution would be to remove the default padding from the `Text` component in `StaTheme` and add padding to the components that contain the text. However that is quite a refactor as it it used throughout the app. Better to do this in a separate ticket? 

Aside from that I have now replaced the `Center` component with `VStack` and `alignItems center` as suggested anyway, as this is probably less confusing and in line with other pages.

Reply:
Ok yeah makes sense, maybe it would be better to change the `Text` default padding in the theme file.  If you're not doing that here, are you ok to create a separate ticket for it so we don't lose this as a task?

Reply:
Yeah, I think it would be better in a separate ticket, due to the refactor that comes with it and not to hold this one up. We can think of other improvements to the theme perhaps to include in that ticket.